jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/TimelinesTests.java

```diff
@@ -136,15 +142,19 @@ public void oppositeTimeLineTest(int index, String date, Colors color) {
         TimeLineItem<DecadeEvent, UIElement> item = oppositeTimeLine.item(index);
         item.has().dotColor(color);
         item.opposite().has().text(date);
-        item.opposite().has().css("color", color.value());
+        item.is().oppositeColor( color.value());
+        item.oppositeTitle().has().text(LOREM_IPSUM_TITLE);
```

Comment:
item.oppositeTitle().has().text(LOREM_IPSUM_TITLE) -> item.opposite().title().has().text(LOREM_IPSUM_TITLE);
Please create appropriate methods for that